### PR TITLE
Handle cases where no topics are available on the account

### PR DESCRIPTION
**Pull request by brunoluiz**

### CODE REVIEW

File: src/Messenger/Clients/Sns.php [hunk 1]
```diff
@@ -67,6 +73,18 @@ private function getNameFromArn($arn)
         return $name;
     }
 
+    private function getDataToPublish($topic, $message)
+    {
+        //Up to 256KB of Unicode text.
+        $messageEncoded = json_encode($message);
+
+        return [
+            'TopicArn' => $this->getArnFromName($topic),
```
Comment (guilhermesianilinx) [inline src/Messenger/Clients/Sns.php:82]: Why did u name vars in PascalCase?

Comment (guilhermesianilinx) [reply src/Messenger/Clients/Sns.php:82]: oo srry! I saw why below

Comment (brunoluiz) [reply src/Messenger/Clients/Sns.php:82]: The AWS SDK always use pascal case for its settings (as you can see, this method is only called on the publish method)

File: src/Messenger/Clients/Sns.php [hunk 2]
```diff
@@ -151,24 +169,23 @@ public function unsubscribe(string $topic, string $subscriptionId): bool
...
 
             $this->createTopic($topic);
-            $this->snsClient->publish($publishData);
+
+            $data = $this->getDataToPublish($topic, $message);
+            $this->snsClient->publish($data);
+        } catch (NoResourceFoundException $e) {
+            $this->createTopic($topic);
+
+            $data = $this->getDataToPublish($topic, $message);
```
Comment (guilhermesianilinx) [inline src/Messenger/Clients/Sns.php:187]: This part is duplicated and can be out of catch block and before return, Can't?

**Review by jhozefem — APPROVED** [review]
LGTM

**Review by flaviozantut — APPROVED** [review]
LGTM

**Pull request merged**